pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -950,7 +950,15 @@ const (
 	ProxyModeIPVS ProxyMode = "IPVS"
 )
 
-// KubeletConfig contains configuration settings for the kubelet.
+// ContainerLogSize determines container log rotation in kubelet
+type ContainerLogSize string
+
+const (
+	// DefaultContainerLogMaxSize sets default 100Mi as default for ContainerLogMaxSize
+	DefaultContainerLogMaxSize ContainerLogSize = "100Mi"
+)
+
+// KubeletConfig contains configuration settings for the kubelet
```

Comment:
Can you revert the change on this doc string?

```suggestion
// KubeletConfig contains configuration settings for the kubelet.
```